README.md

```diff
@@ -109,7 +107,3 @@ Artifacts can be browsed via the web interface, for example, the latest in recon
 <a href="https://eicweb.phy.anl.gov/api/v4/projects/400/jobs/artifacts/master/raw/results/dvcs/Q2.png?job=dvcs:results">
 <img src="https://eicweb.phy.anl.gov/api/v4/projects/400/jobs/artifacts/master/raw/results/dvcs/Q2.png?job=dvcs:results" width="600px" />
```

Comment:
I'm puzzled by this (and image above) not working. Any help appreciated.

Reply:
https://docs.gitlab.com/ee/ci/jobs/job_artifacts.html#from-a-url

Reply:
I found a recent docs:results job (id: 2418810, 2418696) and the results directory doesn't contain any png's.. just a root file.. 

I chose a different job name and grabbed an image file and this link downloads the image to my laptop but renders in an online html editor.
https://eicweb.phy.anl.gov/api/v4/projects/400/jobs/artifacts/master/raw/results/single/truth_reconstruction/Electrons_e-_1GeV_45to135deg.png?job=single:results